tests/apps/Dockerfile-windows

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License.
 # ------------------------------------------------------------
 
-FROM golang:1.15-nanoserver-1809
+FROM mcr.microsoft.com/windows/nanoserver:2004
```

Comment:
Will this run in AKS? I remember trying to use a newer version and getting a version mismatch.

Reply:
oh good to know. I will change to 1809.